src/spec/integration/product-type-importer.spec.coffee

```diff
@@ -93,6 +94,25 @@ describe 'ProductTypeImporter', ->
     .then (res) ->
       expect(res.body.results.length).to.equal 1
 
+  it 'should import product type and generate key', ->
+    key = "unslugified name 1928 - "
+
+    sphereClient.productTypes.fetch()
+    .then (res) ->
+      expect(res.body.results.length).to.equal 0
+      console.log "Importing product type using importer"
+
+      testProductType.name = key
+      delete testProductType.key
+
+      importer.import {productTypes: [testProductType]}
+    .then ->
+      sphereClient.productTypes.fetch()
+    .then (res) ->
+      expect(res.body.results.length).to.equal 1
+      productType = res.body.results[0]
+      expect(productType.key).to.equal _.slugify(key)
```

Comment:
This test was meant to verify that import script will create default key when it is not provided.. How would you do that test?